kafka-rest/src/test/java/io/confluent/kafkarest/resources/v3/ProducerMetricsTest.java

```diff
@@ -78,6 +79,7 @@ public void testRateMetrics() throws Exception {
         .forEach(
             n -> {
               metrics.recordError();
+              metrics.recordRateLimited();
```

Comment:
Just a thought, rather than a requirement.  Is this test not very good any more, now that there are more metrics? 

I'm wondering if we should test the metrics one at a time.

I'm thinking about the case where the non-rate limiting accidentally triggers the rate limiting metric too. But then the rate limiting would have to not trigger for the test to pass....

Reply:
Ack.